src/_includes/layouts/event.html

```diff
@@ -79,6 +79,24 @@ <h1>{{ title }}</h1>
 					</span>
 				</p>
 			</main>
+			<div id="sesh" aria-hidden="true">
+				<style>
+					#sesh:not(:target) {
```

Comment:
Can we pull these styles out into one of our stylesheets?

Reply:
we definitely can. i had put it here just because it's new and this is the only page/layout that will be sure to use the functionality. moving it up the chain kinda suggests that it might be more "globally legitimized"

but it's not a holdfast opinion for me

Reply:
Ah, makes sense. Given that, I wouldn't worry about this too much then.

Reply:
i've implemented the spirit of your feedback in a slightly different way.
to better isolate the code, i've moved it to a partial. that partial as a single `id` which the inline-stylesheet uses as a scope for all selectors.

Reply:
Ah! I love making this a partial! What a great idea!